misaligned.py

```diff
@@ -1,13 +1,18 @@
-
+Reference_Manual_List = []
 def print_color_map():
     major_colors = ["White", "Red", "Black", "Yellow", "Violet"]
     minor_colors = ["Blue", "Orange", "Green", "Brown", "Slate"]
     for i, major in enumerate(major_colors):
         for j, minor in enumerate(minor_colors):
-            print(f'{i * 5 + j} | {major} | {minor}')
+            Concate_ReferenceManual(i,j,major,minor)         
+            
     return len(major_colors) * len(minor_colors)
 
+def Concate_ReferenceManual(i,j,major,minor):
+    global Reference_Manual_List
+    Reference_Manual_List.append(f'{i * 5 + j} | {major} | {minor}')
 
 result = print_color_map()
 assert(result == 25)
+assert(Reference_Manual_List[0].find("|") == Reference_Manual_List[24].find("|"))

```

Comment:
what is the intention of this assert? 0th item same as 24th item?
Intention is to write test test cases that test correct outcome. Whether production code has implementation or not is secondary. Test cases need to be defined first.

Reply:
@prithamvshetty Hi, 

This is for to check misalignment. 

In 0th row, values is like       " 0 | Black | Green " ,
But in 24th row is like          " 24 | Green | Pink"

this "|" is shifted because of 24 has 2 counts. If the seperate is in 0th and 24th are in same position, We can assume that table is properly aligned. So in that perspective only i did this

Reply:
Understood 👍 thanks for the clarification... Also try to verify the content of each pair. Without that it is still incomplete 😊